packages/cli/src/databases/dsl/index.ts

```diff
@@ -7,11 +7,15 @@ export const createSchemaBuilder = (tablePrefix: string, queryRunner: QueryRunne
 	column: (name: string) => new Column(name),
 	/* eslint-disable @typescript-eslint/promise-function-async */
 	// NOTE: Do not add `async` to these functions, as that messes up the lazy-evaluation of LazyPromise
-	createTable: (name: string) => new CreateTable(name, tablePrefix, queryRunner),
-	dropTable: (name: string) => new DropTable(name, tablePrefix, queryRunner),
-	createIndex: (name: string, tableName: string, columnNames: string[], isUnique = false) =>
-		new CreateIndex(name, tableName, columnNames, isUnique, tablePrefix, queryRunner),
-	dropIndex: (name: string, tableName: string) =>
-		new DropIndex(name, tableName, tablePrefix, queryRunner),
+	createTable: (tableName: string) => new CreateTable(tableName, tablePrefix, queryRunner),
+
+	dropTable: (tableName: string) => new DropTable(tableName, tablePrefix, queryRunner),
+
+	createIndex: (tableName: string, columnNames: string[], isUnique = false) =>
```

Comment:
can we add the `indexName` as a last optional param, and generate the index name only if not is pass? 
we'll need that needed to convert older migrations like [this one](https://github.com/netroy/n8n/blob/master/packages/cli/src/databases/migrations/mysqldb/1611149998770-AddWebhookId.ts#L14)